CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG for mcrouter
 
+## [v3.0.0] (2017-08-15)
+
+### Breaking changes:
+* Removed support for Ubuntu 14.04
+* Converted `node['folly']['version']` to `node['mcrouter']['commit_hash']` given that FB has a weird release process.
```

Comment:
What's weird about the release process?
Looking at https://github.com/facebook/folly/releases it's appears to be weekly release. By removing version, and not having a hash for the `folly` repo, does that mean it's getting HEAD? Typically not advisable unless good reason.